lib-injection/src/test/resources/end-to-end/helm-values.yaml

```
@@ -0,0 +1,23 @@
+targetSystem: "linux"
+clusterAgent:
+  enabled: true
+  image:
+    name: ""
+    tag: ahmed-auto-instru
```

```suggestion
    tag: master
```
The code is merged, you don't have to pull my custom branch.

---

Changed in the template in `functions.sh`